vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/certs/single_certkey_pair.go

```diff
@@ -97,6 +97,8 @@ func (c *DynamicCertKeyPairLoader) CheckCerts() error {
 	c.currentValue.Store(newRuntimeConfig)
 	c.currentContent = newContent // this is single threaded, so we have no locking issue
 
+	glog.V(2).Infof("Detected certificate change: %q: %s, %q: %s", c.fileReference.Cert, newContent.CertSHA256(), c.fileReference.Key, newContent.KeySHA256())
```

Comment:
I think this is worthy of info.  But you should make the output match must-gather.